app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -9,6 +9,11 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
     $scope.yearMoinsUn = moment($scope.situation.dateDeValeur).subtract('years', 1).format('YYYY');
     $scope.currentMonth = moment($scope.situation.dateDeValeur).format('MMMM YYYY');
 
+    // Pour les Auto-entrepreneurs
+    $scope.onGoingLabel = 'J\'aurai un chiffre d\'affaires non nul en ' + $scope.currentMonth;
```

Comment:
When your string contains apostrophes, either use the typographically-correct rounded apostrophe symbol (`’`), or embed the string in double quotes to avoid escaping your single quotes.

In JS, single and double quotes are interchangeable. It is common to use single quotes to look like languages that do interpolation with double quotes, but escaping is worse.